### PR TITLE
Using @bind

### PR DESCRIPTION
**Pull request by jakearchibald**
I should have done this before merging. Please forgive my trespasses.

### CODE REVIEW

File: src/lib/PointerTracker/index.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { bind } from '../util';
```
Comment (surma) [inline src/lib/PointerTracker/index.ts]: Just for me: We are doing extension-less imports because webpack, right?

Comment (jakearchibald) [reply src/lib/PointerTracker/index.ts]: A vscode extension I used added this import for me 😄.

Not wedded to extension-less imports. Any feels here @developit @kosamari?

Comment (surma) [reply src/lib/PointerTracker/index.ts]: FWIW I tend to use `.js` extension. TypeScript automatically prefers `.ts` files with the same name if they exist. But no strong feels on my side.

Comment (developit) [reply src/lib/PointerTracker/index.ts]: TBH it doesn't matter to me. It's extremely uncommon to use extensions in Webpack, though.

Comment (mathiasbynens) [reply src/lib/PointerTracker/index.ts]: Drive-by opinion: explicit > implicit

Comment (surma) [reply src/lib/PointerTracker/index.ts]: >  It's extremely uncommon to use extensions in Webpack

Probably the strongest argument so far, considering one of our goals was to meet developers where they are currently at.

**Pull request merged**